static/app/views/projectInstall/issueAlertOptions.tsx

```diff
@@ -51,6 +52,10 @@ type StateUpdater = (updatedData: RequestDataFragment) => void;
 type Props = AsyncComponent['props'] & {
   onChange: StateUpdater;
   organization: Organization;
+  alertSetting?: string;
+  interval?: string;
+  metric?: MetricValues;
+  threshold?: string;
```

Comment:
these new props are required for when the selected alert option is customized. If users have previously selected the customized option, all defined values will be populated by default